javaparser-core-generators/src/main/java/com/github/javaparser/generator/core/CoreGenerator.java

```diff
@@ -71,7 +71,6 @@ private void run(SourceRoot sourceRoot, SourceRoot generatedJavaCcSourceRoot) th
         new CloneGenerator(sourceRoot).generate();
         new GetMetaModelGenerator(sourceRoot).generate();
         new MainConstructorGenerator(sourceRoot).generate();
-        new FinalGenerator(sourceRoot).generate();
```

Comment:
Maybe make it a non-final generator? The generator is there for consistency, so we can easily turn "add" into "remove", I think.